src/test/videopress/edit.js

```diff
@@ -38,6 +45,8 @@ describe( 'VideoPress block', () => {
 		// Add block
 		await addBlock( screen, 'VideoPress' );
 
+		jest.runOnlyPendingTimers();
+
```

Comment:
When running the command `TEST_RN_PLATFORM=ios npm run test src/test/videopress/edit.js` I'm getting the following warning:
```
 A function to advance timers was called but the timers APIs are not mocked with fake timers. Call `jest.useFakeTimers({legacyFakeTimers: true})` in this test file or enable fake timers for all tests by setting {'enableGlobally': true, 'legacyFakeTimers': true} in Jest configuration file.
    Stack Trace:

          46 | 		await addBlock( screen, 'VideoPress' );
          47 |
        > 48 | 		jest.runOnlyPendingTimers();
             | 		     ^
          49 |
          50 | 		// Get block
          51 | 		const videoPressBlock = await getBlock( screen, 'VideoPress' );

          Error:
          at FakeTimers._checkFakeTimers (node_modules/@jest/fake-timers/build/legacyFakeTimers.js:394:13)
          at FakeTimers.runOnlyPendingTimers (node_modules/@jest/fake-timers/build/legacyFakeTimers.js:189:10)
          at Object.runOnlyPendingTimers (src/test/videopress/edit.js:48:8)
          at asyncGeneratorStep (node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
          at _next (node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
          at tryCallOne (gutenberg/node_modules/promise/lib/core.js:37:12)
          at gutenberg/node_modules/promise/lib/core.js:123:15
          at flush (gutenberg/node_modules/asap/raw.js:50:29)
```

Probably we could use the `withFakeTimers` helper to address the issue:
```
// When the block is inserted, it automatically opens the media picker.
// On iOS, this picker is displayed using a timer, so we need to run it
// to allow any DOM update.
await withFakeTimers( () => jest.runOnlyPendingTimers() );
```